src/main/java/com/ericsson/ei/controller/SubscriptionControllerImpl.java

```diff
@@ -79,6 +79,7 @@
             try {
                 LOG.debug("Subscription creation has been started: " + subscriptionName);
                 subscriptionValidator.validateSubscription(subscription);
+				subscriptionValidator.validateWithSchema(subscription);
```

Comment:
validateSubscription should call the validateWithSchema.

Please make all the methods tobe static in SubscriptionValidator. Remove instantiation of this class here too.

Reply:
done